src/infrastructure/repositories/transaction.typeprm.repo.ts

```
@@ -29,4 +29,71 @@ export class TransactionTypeOrmRepository implements TransactionRepository {
+        account_ids?: string[],
+        type?: string,
+        start_date?: Date,
+        end_date?: Date
+    ): Promise<Array<Transaction>> {
+        let return_trx: Array<Transaction> = [];
+        let all_trx;
+
+        // TODO: to be implemented later
+        if (type) return [];
+
+        if (account_ids && account_ids.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)